### DISCUSSION TITLE
A toggle that makes the cursor change its color based on combo color

### DISCUSSION THREAD

**Discussion by Hiviexd** [ppy/osu#16857] [Ideas]
As shown in this [FA Showcase video](https://osu.ppy.sh/home/news/2022-02-12-new-featured-artist-misoilepunch), it would be a really cool addition to make this a thing in lazer, where your cursor can change color as the combo color changes, and turn grey/white during breaks + it doesn't really seem like a very complicated thing to implement

**Comment by bdach** [comment]
should not be a toggle. maybe will be offered as a skinning option in a new skinning system.
